server/api/calendar.go

```diff
@@ -17,3 +17,50 @@ func (api *api) ViewCalendar(from, to time.Time) ([]*remote.Event, error) {
 
 	return client.GetUserDefaultCalendarView(api.user.Remote.ID, from, to)
 }
+
+func (api *api) CreateCalendar(calendar *remote.Calendar) (*remote.Calendar, error) {
+	client, err := api.MakeClient()
+	if err != nil {
+		return nil, err
+	}
+
+	return client.CreateCalendar(calendar)
+}
+
+func (api *api) CreateEvent(calendarEvent *remote.Event) (*remote.Event, error) {
+	client, err := api.MakeClient()
+	if err != nil {
+		return nil, err
+	}
+
+	return client.CreateEvent(calendarEvent)
+}
+
+func (api *api) DeleteCalendar(calendarID string) error {
+	client, err := api.MakeClient()
+	if err != nil {
+		return err
+	}
+
+	return client.DeleteCalendar(calendarID)
+}
+
+func (api *api) FindMeetingTimes(meetingParams *remote.FindMeetingTimesParameters) (*remote.MeetingTimeSuggestionResults, error) {
+	client, err := api.MakeClient()
+	if err != nil {
+		return nil, err
+	}
+
+	return client.FindMeetingTimes(meetingParams)
+}
+
+func (api *api) GetUserCalendars(userID string) ([]*remote.Calendar, error) {
```

Comment:
s/userID/mattermostUserID/

Reply:
Also, I don't see the ID being used, so maybe this should be `GetMyCalendar() ...`?